File: Backend/src/main/java/com/ssafy/closetory/dto/post/PostDetailResponse.java
package com.ssafy.closetory.dto.post;

import java.time.LocalDateTime;
import java.util.List;

public record PostDetailResponse(
    Integer postId,
    String title,
    String photoUrl,
    String content,
    List<PostItemResponse> items,
    LocalDateTime createdAt,
    Integer views,
    Integer likeCount,
    Boolean isLiked,
    Integer userId,
    String nickname,
    String profilePhotoUrl) {}

File: FrontEnd/app/src/main/java/com/ssafy/closetory/dto/PostDetailResponse.kt
package com.ssafy.closetory.dto

data class PostDetailResponse(
    val postId: Int,
    val userId: Int,
    val nickname: String,
    val profilePhotoUrl: String?,
    val title: String,
    val photoUrl: String?,
    val content: String,
    val items: List<PostDetailItemDto> = emptyList(), // 기본적으로 빈 리스트 반환
    val createdAt: String, // 시간의 경우 String으로 받는 것이 안전.
    val views: Int,
    val likeCount: Int,
    val isLiked: Boolean
)

data class PostDetailItemDto(val clothesId: Int, val photoUrl: String, val isSaved: Boolean)
